### DISCUSSION TITLE
Bind kill to button1 for specific application but do not affect others

### DISCUSSION THREAD

**Discussion by jose1711** [i3/i3#5286] [Q&A] [answered]
When I put the following into config
```
bindsym --whole-window button1 [class="meteo-qt"] kil
```
it works as intended for `meteo-qt` application (click on application window closes it) but at the same time all other applications stop processing left mouse buttons. Can someone please advise how to fix?

**Comment by Airblader** [comment, marked as answer]
This binding does _not_ do what you expect. What it does do is this: anytime you click that mouse button on any window, all meteo-qt instances will be closed (no matter which window you clicked on). A way to restrict it would be to also add `con_id=__focused__` to the criteria, but that will not resolve your question.

i3wm does not support any application-specific bindings.

**Comment by jose1711** [comment]
Thank you for your quick reply.

> i3wm does not support any application-specific bindings.

Noted. Is this true for any window manager btw?

**Reply by Airblader** [reply]
I don't know whether (or which) other window managers do support this, sorry. But if you want a click on the entire window to do it, you can probably solve it with standalone tools. There are things like sxhkd and others, not sure if any of those also do mouse bindings, but perhaps even `xdotool` could be made to do this somehow.
